libfoundation/src/foundation-string.cpp

```diff
@@ -6820,6 +6820,25 @@ __MCStringCreateWithStrings(MCStringRef& r_string, bool p_has_separator, unichar
     return t_success;
 }
 
+////////////////////////////////////////////////////////////////////////
+
+#if defined(__WINDOWS__)
+
+MC_DLLEXPORT_DEF bool
+MCStringCreateWithBSTR(const BSTR p_bstr,
+                       MCStringRef& r_string)
+{
+    return MCStringCreateWithBytes(reinterpret_cast<const byte_t*>(p_bstr),
```

Comment:
Would it not be more appropriate to use CreateWithChars here? BSTR's are UTF-16 strings so are actually arrays of uint16_t, not bytes.